src/crashdb/services.py

```diff
@@ -121,7 +121,7 @@ def insert_rows_with_arcpy(self, table_name, rows):
                 try:
                     cursor.insertRow(row)
                 except Exception as e:
-                    self.logger.info('could not insert {} {}', row, e)
+                    self.logger.info('could not insert {} {}'.format(row, e))
```

Comment:
i don't think you want to use format here since the formatting code will execute even if the logging is set to something higher than info. It's a small perf improvement but a good practice to be aware of.